libimage/filters_test.go

```diff
@@ -0,0 +1,69 @@
+package libimage
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFilterNames(t *testing.T) {
+	registries := []string{
+		"localhost",
+		"docker.io",
+		"foo.bar",
+		"registry.access.redhat.com",
+	}
+	names := []string{
```

Comment:
no test for `bar` not matching `localhost/foobar` (as in https://github.com/containers/podman/issues/11905)? :)

Reply:
Added a match for *box to match busybox

Reply:
I was rather thinking about ensuring that with `"localhost/foo:latest"` and `"localhost/foobar:latest`" in `names` a test for `{"bar:*", 1}` succeeds.